.env.defaults

```diff
@@ -57,7 +57,7 @@ SHOW_ADS=true
 ENABLE_MATURE=false
 ENABLE_UI_NOTIFICATIONS=true
 #SHOW_TAGS_INTRO=false
-USE_LOCAL_HOMEPAGE_DATA=true
+USE_LOCAL_HOMEPAGE_DATA=false
```

Comment:
I assume this is temporary and not needed for this PR?

Aside: most likely you already now, but you can create a blank `web/.env` file locally, and add `USE_LOCAL_HOMEPAGE_DATA=false` (and full rebuild, can't hot reload).
This saves the need to commit and revert.  Also works for dev instances (need to ssh in).